TLCGen/Views/Tabs/AlgemeenTab/Tabs/KruispuntArmFaseCyclusViewModel.cs

```diff
@@ -2,87 +2,88 @@
 using TLCGen.Helpers;
 using TLCGen.Models;
 
-namespace TLCGen.ViewModels;
-
-public class KruispuntArmFaseCyclusViewModel : ViewModelBase, IViewModelWithItem
+namespace TLCGen.ViewModels
```

Comment:
waarom hier geen filescoped namespace? zou eigenlijk goed zijn (ná de PR) eens overal te doen

Reply:
erg lullige reden: dat leidde bij mijn (VS 2019) setup tot compileer errors. En in plaats van 2 dagen te zoeken naar de oorzaak, was het sneller om in 5 minuten die namespaces aan te passen.